index.html

```diff
@@ -33,7 +32,120 @@ <h2 class="text-container" id="sdsection">
       </ul>
       <div class="button">
       <img src="arrow-button.svg" class="arrow-button"><a href="#"></a></img>
+      </div>
+    </section>
+    <section class="works">
+      <h1 class="recent-works">My Recent Works</h1>
+      <div class="card-container">
```

Comment:
- [ ] Nice work on this header, however, there are still some details that you missed to add to match the Figma design 🔽 please work on it.

## Your Design
![2023-03-25_172549](https://user-images.githubusercontent.com/104319462/227732517-88d8b423-4c9c-4416-a0df-9a333f275cfc.png)

## Figma Design
![2023-03-25_172832](https://user-images.githubusercontent.com/104319462/227732507-181ff5f3-3b90-413b-b6d8-6a0ee9e71558.png)